.github/workflows/ci_dependabot_sync.yml

```
@@ -0,0 +1,38 @@
+    runs-on:  ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
```

actionlint (1.7.10)

could not parse as YAML: did not find expected key

(syntax-check)